viewslides.py

```diff
@@ -1310,7 +1310,6 @@ def _download_result_cb(self, getter, tempfile, suggested_name, tube_id):
 
         _logger.debug("Got document %s (%s) from tube %u",
                       tempfile, suggested_name, tube_id)
-        self._load_document(tempfile)
```

Comment:
What is your purpose in removing call to _load_document in _download_result_cb?  The commit message doesn't explain.

Reply:
This commit was pushed almost 3 months ago, I don't exactly remember why I have made the change. I'll look through and get back and if it's still required, I'll do that in a separate commit.

Reply:
I have undone the change in ed4d934